components/host-dashboard/HostDashboardExpenses.js

```diff
@@ -213,6 +213,17 @@ const HostDashboardExpenses = ({ hostSlug, isNewAdmin }) => {
           )}
         </DismissibleMessage>
       )}
+      {!loading && data?.host.hasDisputedOrders && (
+        <Box mb={4}>
+          <MessageBox type="warning" withIcon>
+            <FormattedMessage
+              id="host.disputes.warning"
+              defaultMessage="Fraud Warning: There are disputed charges that need review."
+            />{' '}
+            <Link href={`/${hostSlug}/admin/orders?status=DISPUTED`}>Review Disputes</Link>
+          </MessageBox>
+        </Box>
+      )}
```

Comment:
Suggestion: move this to `pages/admin-panel` and make it full-width above all the Grid component.